### PR TITLE
20 notification on lead card

### PR DESCRIPTION
**Pull request by NazarPo**
Implements #20

### CODE REVIEW

File: client/src/modules/lead/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 import { connect } from 'react-redux';
 import LeadModel from '../../../models/Lead';
 import Stage from '../../../models/Stage';
+import { loadFirstActivityInLeadsPlan } from '../EditLead/Activities/activityActions';
 import { loadLeadboard } from '../leadActions';
 import Lead from './Lead/Lead';
 
 interface Props {
   leads: any;
+  activities: any;
```
Comment (SamardakOleh) [inline client/src/modules/lead/Dashboard/Dashboard.tsx]: maybe here should be better name

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    const expectedCountOfLeads = leads.leads._5b6b123391e0774579ed6701.leads.length.toString();
+    expect(wrapper.find('small').text()[0]).toEqual(expectedCountOfLeads);
+  });
+
+  it('check if user have leads in stages it should been rendered Lead components', () => {
+    expect(wrapper.find(Lead)).toHaveLength(6);
+  });
+
+  it('check if lead have planned activity', () => {
+    expect(wrapper.find(Lead).at(0).props().activityStatus).toEqual('Planned');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  it('check if lead have active activity', () => {
+    expect(wrapper.find(Lead).at(1).props().activityStatus).toEqual('Active');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  it('check if lead have overdue activity', () => {
+    expect(wrapper.find(Lead).at(2).props().activityStatus).toEqual('Overdue');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  it('check if lead have overdue today activity with planed time', () => {
+    expect(wrapper.find(Lead).at(3).props().activityStatus).toEqual('Overdue');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  it('check if lead have planed today activity with planed time', () => {
+    expect(wrapper.find(Lead).at(4).props().activityStatus).toEqual('Active');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: client/src/modules/lead/Dashboard/index.test.tsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+  });
+
+  it('check if lead have no activities', () => {
+    expect(wrapper.find(Lead).at(5).props().activityStatus).toEqual('NoActivity');
```
Comment (valentyntu) [inline client/src/modules/lead/Dashboard/index.test.tsx]: use extracted constant

File: technical debt.md [hunk 8]
```diff
@@ -13,5 +13,9 @@ This file contains everything that must be fixed/changed/done. We didn't do that
 - Refactor styles (extract global)
 - Remove redundant requests sending on loading leads
 - Snapshot testing
+<<<<<<< HEAD
```
Comment (valentyntu) [inline technical debt.md]: fix this

**Pull request merged**